server/services/gameLogic.js

```diff
@@ -38,7 +38,7 @@ let cards = [];
 let votes = [];
 
 /* Return the list of players, without their assigned cards */
-exports.getPlayers = () => players.map(player => ({ username: player.username, score: player.score }));
+const getPlayers = exports.getPlayers = () => players.map(player => ({ username: player.username, score: player.score }));
```

Comment:
This is just a stylistic suggestion and you may well disagree.
```suggestion
const getPlayers = () => players.map(player => ({ username: player.username, score: player.score }));
exports.getPlayers = getPlayers;
```